optionals/noactionmenu/CfgActions.hpp

```diff
@@ -1,24 +1,1108 @@
+        showIn3D = 0;
+        showWindow = 0;
+    };
+    class HealBleedingAuto: None {
+        show = 0;
+        showIn3D = 0;
+        showWindow = 0;
+    };
+    class HealBleedingOnly: None {
+        show = 0;
+        showIn3D = 0;
+        showWindow = 0;
+    };
+    class HealBleedingSelfOnly: None {
+        show = 0;
+        showIn3D = 0;
+        showWindow = 0;
+    };
+    class HealSoldier: None {
+        show = 0;
+        showIn3D = 0;
+        showWindow = 0;
+    };
+    class HealSoldierAuto: None {
+        show = 0;
+        showIn3D = 0;
+        showWindow = 0;
+    };
+    class HealSoldierSelf: None {
+        show = 0;
+        showIn3D = 0;
+        showWindow = 0;
+    };
+    class HelicopterAutoTrimOff: None {
+        show = 0;
+        showIn3D = 0;
+        showWindow = 0;
+    };
+    class HelicopterAutoTrimOn: None {
+        show = 0;
+        showIn3D = 0;
+        showWindow = 0;
+    };
+    class HelicopterTrimOff: None {
+        show = 0;
+        showIn3D = 0;
+        showWindow = 0;
+    };
+    class HelicopterTrimOn: None {
+        show = 0;
+        showIn3D = 0;
+        showWindow = 0;
+    };
+    class HideBody: None {
+        show = 0;
+        showIn3D = 0;
+        showWindow = 0;
+    };
+    class HideWeapon: None {
+        show = 0;
+        showIn3D = 0;
+        showWindow = 0;
+    };
+    class HookCargo: None {
+        show = 0;
+        showIn3D = 0;
+        showWindow = 0;
+    };
+    class IngameMenu: None {
+        show = 0;
+        showIn3D = 0;
+        showWindow = 0;
+    };
+    class IRLaserOff: None {
+        show = 0;
+        showIn3D = 0;
+        showWindow = 0;
+    };
+    class IRLaserOn: None {
+        show = 0;
+        showIn3D = 0;
+        showWindow = 0;
+    };
+    class LadderDown: None {
+        show = 0;
+        showIn3D = 0;
+        showWindow = 0;
+    };
+    class LadderOff: None {
+        show = 0;
+        showIn3D = 0;
+        showWindow = 0;
+    };
+    class LadderOnDown: None {
+        show = 0;
+        showIn3D = 0;
+        showWindow = 0;
+    };
+    class LadderOnUp: None {
+        show = 0;
+        showIn3D = 0;
+        showWindow = 0;
+    };
+    class LadderUp: None {
+        show = 0;
+        showIn3D = 0;
+        showWindow = 0;
+    };
+    class Land: None {
+        show = 0;
+        showIn3D = 0;
+        showWindow = 0;
+    };
+    class LandGear: None {
+        show = 0;
+        showIn3D = 0;
+        showWindow = 0;
+    };
+    class LandGearUp: None {
+        show = 0;
+        showIn3D = 0;
+        showWindow = 0;
+    };
+    class LightOff: None {
+        show = 0;
+        showIn3D = 0;
+        showWindow = 0;
+    };
+    class LightOn: None {
+        show = 0;
+        showIn3D = 0;
+        showWindow = 0;
+    };
+    class ListLeftVehicleDisplay: None {
+        show = 0;
+        showIn3D = 0;
+        showWindow = 0;
+    };
+    class ListPrevLeftVehicleDisplay: None {
```

Comment:
Also, some of these inheritance trees are wrong. `ListPrevLeftVehicleDisplay` inherits from `ListPrevRightVehicleDisplay`, not "`None`". Just as an example. Don't feel like checking all of these.

Reply:
Added some classes that I missed and corrected for this.